x-pack/plugins/cases/server/client/cases/push.ts

```diff
@@ -40,6 +48,30 @@ function shouldCloseByPush(
   );
 }
 
+const changeAlertsStatusToClose = async (
+  caseId: string,
+  caseService: CasesClientArgs['caseService'],
+  alertsService: CasesClientArgs['alertsService']
+) => {
+  const alertAttachments = (await caseService.getAllCaseComments({
+    id: [caseId],
+    options: {
+      filter: nodeBuilder.is(`${CASE_COMMENT_SAVED_OBJECT}.attributes.type`, CommentType.alert),
+    },
+  })) as SavedObjectsFindResponse<CommentRequestAlertType>;
+
+  await alertsService.updateAlertsStatus(
```

Comment:
does each of this loop create a new request? if so:

1) can we use a batch api for this? 
2) use `p-map` to set a concurrency level instead of firing all the alerts at the same time?

What happens in cases where a case have hundreds of alerts? does the user wait here for minutes while all these requests finish?

Reply:
Thank you, Esteban! No, the loop formats the data. One request is being made to ES no matter the number of alerts.

Reply:
That makes more sense.

Can you split this in two lines? one where it creates a variable with the payload and another one with the call?
that makes it easier to read.